### DISCUSSION TITLE
Option to enable volume control in song select

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#21581] [Ideas] [answered]
I would like an option to enable volume control in song select, just like the option to disable it while gameplay.
Example:
![image](https://user-images.githubusercontent.com/88291316/206490884-b37d6853-e3af-44f7-8ae1-493dc630ae8e.png)

Because song often have different volumes, and it's really annoying to get earraped :)
I know you can use the little icon in the top right (which is a AWESOME detail), but it's way harder to use the little rectangle than just moving your mouse in a little empty space.

**Comment by ghost** [comment, marked as answer]
You can use alt+scroll wheel. Pretty sure this behaviour matches stable
